.github/label-actions.yml

```diff
@@ -27,6 +27,22 @@ There are two types of logs to collect:
 
 - This will open the log folder locally. Please zip up this folder and attach it to the issue."
 
+# actions for Needs Logs - Azure label
+Needs Logs - Azure:
+  comment: "We need more info to debug your Azure Active Directory issue. If you could attach your logs to the issue (ensure no private data is in them), it would help us fix the issue much faster.
+
+- In the settings menu, find the setting titled `Azure: Logging Level` and select the `Verbose` option
+
+- Run the process that produces your error
+
+- Open command palette (Click **View** -> **Command Palette**)
+
+- Run the command: **`Developer: Open Logs Folder`**
+
+- Follow this path to find the Azure Accounts log file: `[default log folder]/exthost1/output_logging_[earliest timestamp]/2-Azure Acounts.log`
```

Comment:
`#-Azure Accounts.log` - the number isn't guaranteed to be 2 (it's based on the order the extension registers its output channel)